sdk/keyvault/tests-jca.yml

```
@@ -0,0 +1,51 @@
+trigger: none
+
+extends:
+  template: /eng/pipelines/templates/stages/archetype-sdk-tests.yml
+  parameters:
+    ServiceDirectory: keyvault
+    TestResourceDirectories:
+      - keyvault/test-resources-jca
+    TimeoutInMinutes: 240
+    SupportedClouds: 'Public,UsGov,China,Canary'
+    EnvVars:
+      AZURE_LOG_LEVEL: 2
+    CloudConfig:
+      Public:
+        SubscriptionConfiguration: $(sub-config-azure-cloud-test-resources)
```

Do we need this here too?
```suggestion
        ServiceConnection: azure-sdk-tests
        SubscriptionConfigurationFilePath: eng/common/TestResources/sub-config/AzurePublicMsft.json
        SubscriptionConfiguration: $(sub-config-azure-cloud-test-resources)
```

---

This is not needed here, will remove it.

---

Oh, it was a suggestion, yeah this change is not needed here.